docs/reference/kusk-cli.md

```diff
@@ -0,0 +1,206 @@
+# Kusk CLI
+
+Kusk is a CLI tool designed to help you manage common tasks required when
+running Kusk Gateway.
+
+Currently we support the following commands:
+- `install` - installs Kusk Gateway and all its components with a single command. (Requires a helm installation)
+- `api generate` - for creating Kusk Gateway API resources from your OpenAPI specification document.
+
+---
+
+# Table of contents
```

Comment:
Not sure you need this table of contents, mkdocs generates one automatically 
![image](https://user-images.githubusercontent.com/27779735/165133045-5a30c9f9-9ebd-4c4b-b55f-c6397a8b7e50.png)